src/rgw/support/rgw_support_bundle

```diff
@@ -179,6 +201,32 @@ class RGWSupportBundle:
         RGWSupportBundle._generate_tar(bundle_id, target_path)
         RGWSupportBundle._cleanup()
 
+    @staticmethod
+    def _get_remaining_folder_size(directory:str, size_limit:str) -> str:
```

Comment:
calculate the remaining size of the folder by current size_limit - current_size and return "0B" if the current size is already above